src/diffusers/models/cross_attention.py

```diff
@@ -497,7 +511,7 @@ def __call__(self, attn: CrossAttention, hidden_states, encoder_hidden_states=No
 
         # the output of sdp = (batch, num_heads, seq_len, head_dim)
         hidden_states = F.scaled_dot_product_attention(
-            query, key, value, attn_mask=attention_mask, dropout_p=0.0, is_causal=False
+            query, key, value, attn_mask=attention_mask, dropout_p=0.0, is_causal=False, scale=attn.scale
```

Comment:
Is this backwards compatible?

Reply:
yes since if `scale=None`, the default scale is used ie. the 1/sqrt(D) but only works in 2.1 nightly